web/js/palettes/palettes.legend.js

```diff
@@ -1,10 +1,11 @@
-/**
- * @module wv.palettes
- */
-var wv = wv || {};
-wv.palettes = wv.palettes || {};
+import $ from 'jquery';
+import each from 'lodash/each';
+import first from 'lodash/first';
+import last from 'lodash/last';
+import util from '../util/util';
+import palettes from '../palettes/palettes';
 
-wv.palettes.legend = wv.palettes.legend || function (spec) {
+export function palettesLegend(spec) {
```

Comment:
What do you think about keeping the name `legend`, importing this into the `palettes` parent, and attaching it to the object that exports?